### ISSUE TITLE
MethodScript.com Accessibility - Color Contrast is insufficient

### ISSUE THREAD

**Issue by LeigerGaming** [EngineHub/CommandHelper#1269]
According to a contrast checking tool, the links may be difficult to see or read for someone with low vision.

Passed checks that I looked at:
- Normal dark grey text on white background is OK
- The black search icon on dark orange background for the search button is also OK

Failed check (links):
- Light green links on white background

Light green links:
<img width="300" alt="" src="https://user-images.githubusercontent.com/71436104/110761563-ca471880-829f-11eb-8862-152cff330349.png">

Here's an example:
<img width="300" alt="" src="https://user-images.githubusercontent.com/71436104/110761684-f2367c00-829f-11eb-9882-6d5fef487ef1.png">

References:
- https://webaim.org/resources/contrastchecker/

**Comment by LeigerGaming** [comment]
Looks like the `<strong>` text in the "Dark" theme is also an issue.

There is not enough contrast between that text (#888888) and the background colour (#383838).

<img width="770" alt="" src="https://user-images.githubusercontent.com/71436104/110767177-e0f06e00-82a5-11eb-8198-cc738b9ca41d.png">

References:
- https://webaim.org/resources/contrastchecker/
